azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/CallingViewModel.kt

```diff
@@ -226,6 +230,14 @@ internal class CallingViewModel(
                 canShowLobby(state.localParticipantState.localParticipantRole, state.pipState.status)
             )
 
+            upperMessageBarNotificationLayoutViewModel.update(
+                state.callDiagnosticsState
+            )
+
+            toastNotificationViewModel.update(
+                state.callDiagnosticsState
+            )
```

Comment:
please also verify on screen rotate, going to background and coming back the banners visible